test/parallel/test-bootstrap-modules.js

```diff
@@ -1,14 +1,20 @@
-/* eslint-disable node-core/required-modules */
-
+// Flags: --expose-internals
 'use strict';
 
-// Ordinarily test files must require('common') but that action causes
-// the global console to be compiled, defeating the purpose of this test.
-// This makes sure no additional files are added without carefully considering
-// lazy loading. Please adjust the value if necessary.
-
+// This list must be computed before we require any modules to
+// to eliminate the noise.
 const list = process.moduleLoadList.slice();
 
+require('../common');
 const assert = require('assert');
 
-assert(list.length <= 56, list);
+// We use the internals to detect if this test is run in a worker
+// to eliminate the noise introduced by --experimental-worker
+const { internalBinding } = require('internal/test/binding');
+const { threadId } = internalBinding('worker');
+const isMainThread = threadId === 0;
```

Comment:
`common.isMainThread`?

Reply:
@richardlau Didn't know about this, thanks!